nym-api/src/support/http/mod.rs

```diff
@@ -52,33 +63,26 @@ pub(crate) async fn setup_rocket(
         "/network" => network_routes(&openapi_settings),
         "/api-status" => api_status_routes(&openapi_settings),
         "/ecash" => ecash::routes_open_api(&openapi_settings, config.coconut_signer.enabled),
-        "" => nym_node_routes(&openapi_settings),
+        "" => nym_node_routes_deprecated(&openapi_settings),
```

Comment:
just for pure curiosity, why did you rename those to deprecated? I mean you're not wrong, because I did just deprecate them on my own branch, but how did you know : )